conda/environments/builddocs_py37.yml

```diff
@@ -16,6 +16,5 @@ dependencies:
 - pandoc=<2.0.0
 - pip
 - psutil
-- libhwloc
```

Comment:
1. Can this filename be updated? We don't currently support Python 3.7 any longer
2. If `libhwloc` is being removed here, can we also remove it from the `integration` repo below? It doesn't seem that any other RAPIDS libraries depend on it.

- https://github.com/rapidsai/integration/blob/887942b09b72e081da56ca057197d1b55d5780c6/conda/recipes/rapids-build-env/meta.yaml#L91
- https://github.com/search?q=org%3Arapidsai+libhwloc&type=code

Reply:
> 1. Can this filename be updated? We don't currently support Python 3.7 any longer

Filename updated in https://github.com/rapidsai/ucx-py/pull/829/commits/37c14fb540759d75cd9ea84cc95732320b39edb2 .

> 2. If `libhwloc` is being removed here, can we also remove it from the `integration` repo below? It doesn't seem that any other RAPIDS libraries depend on it.
> 
> * https://github.com/rapidsai/integration/blob/887942b09b72e081da56ca057197d1b55d5780c6/conda/recipes/rapids-build-env/meta.yaml#L91
> * https://github.com/search?q=org%3Arapidsai+libhwloc&type=code

Yes, it absolutely can be removed. I see a few other cases there, namely `cloud-ml-examples` and `node`, I'm not familiar with either of those repositories, but I believe the reason for them listing `libhwloc` was also due to the UCX-Py dependency. Should those be removed as well?

Reply:
> Yes, it absolutely can be removed. I see a few other cases there, namely cloud-ml-examples and node, I'm not familiar with either of those repositories, but I believe the reason for them listing libhwloc was also due to the UCX-Py dependency. Should those be removed as well?

I'm not sure if `cloud-ml-examples` is still maintained. I'll leave that up to you for those repos. But at the very least, we should remove it from the `integration` repo.

Reply:
Since I don't know anything about those projects, I'll leave them as is. Filed https://github.com/rapidsai/integration/pull/437 to remove `libhwloc` support from the integration repository.